library/ssl_tls13_server.c

```diff
@@ -239,6 +190,70 @@ static int ssl_tls13_offered_psks_check_binder_match( mbedtls_ssl_context *ssl,
     return( SSL_TLS1_3_OFFERED_PSK_NOT_MATCH );
 }
 
+MBEDTLS_CHECK_RETURN_CRITICAL
+static int ssl_tls13_psk_external_check_ciphersuites( mbedtls_ssl_context *ssl,
```

Comment:
Will do it in #6167. But I am not sure if it can re-use this function now, I also want to share the routine. For resumption PSK(ticket), it has a field for ciphersuite, we only need check if the ciphersuite in client offered list. That's different with external PSK.

Reply:
I am merge codes in #6167 to here.